src/h2_connection.erl

```diff
@@ -506,20 +507,22 @@ route_frame(F={H=#frame_header{
                                [Conn#connection.type, StreamId, L]),
                     h2_frame_window_update:send(Conn#connection.socket,
                                                 L, StreamId),
-                    send_window_update(self(), L);
+                    send_window_update(self(), L),
+                    recv_data(Stream, F),
+                    {next_state,
+                     connected,
+                     Conn};
```

Comment:
This is really `+ L - L`, but we know math, so we can appreciate that this is really just doing nothing